src/transmission.js

```diff
@@ -77,7 +77,7 @@ class BatchEndpointAggregator {
     let numEncoded = 0;
     let encodedEvents = events.reduce((acc, ev) => {
       try {
-        let encodedEvent = ev.toJSON(); // directly call toJSON, not JSON.stringify, because the latter wraps it in an additional set of quotes
+        let encodedEvent = JSON.stringify(ev);
```

Comment:
not blocking, but is it worth adding a pretty print type thing here? `JSON.stringify(ev, null, '\t')`

Reply:
I don't think so - this is meant to be machine readable more than human readable - one line per event.

Reply:
oh, nevermind - this isn't even something humans will see.  The BatchEndpointAggregator is responsible for building up json to hit /1/batch.  no reason to format it.